services/core-api/app/api/incidents/models/mine_incident.py

```diff
@@ -248,16 +247,48 @@ def validate_reported_timestamp(self, key, reported_timestamp):
         return reported_timestamp
 
     def send_incidents_email(self):
-        recipients = [INCIDENTS_EMAIL, MDS_EMAIL]
-
-        subject = f'Incident Notification for {self.mine_table.mine_name}'
-        body = f'<p>{self.mine_table.mine_name} (Mine no: {self.mine_table.mine_no}) has reported an incident in MineSpace.</p>'
-        body += f'<p>Incident type(s): {", ".join(element.description for element in self.categories)}'
-        body += f'<p><b>Incident information: </b>{self.incident_description}</p>'
-
-        link = f'{Config.CORE_PRODUCTION_URL}/mine-dashboard/{self.mine.mine_guid}/oversight/incidents-and-investigations'
-        body += f'<p>View updates in Core: <a href="{link}" target="_blank">{link}</a></p>'
-        EmailService.send_email(subject, recipients, body)
+        emli_recipients = [INCIDENTS_EMAIL]
+        cc = [MDS_EMAIL]
+        minespace_recipients = [self.reported_by_email]
+        duration = self.reported_timestamp - self.incident_timestamp
+        duration_in_s = duration.total_seconds()
+        days = divmod(duration_in_s, 86400)
+        hours = divmod(days[1], 3600)
+
+        emli_body = open("app/templates/email/incident/emli_incident_email.html", "r").read()
+        minespace_body = open("app/templates/email/incident/minespace_incident_email.html", "r").read()
```

Comment:
Thinking this is going to be reuse lots of times. Do you think we can include email path in config.py?